ui/app/pages/keychains/index.scss

```diff
@@ -70,67 +70,41 @@
 
 .import-account {
   &__title {
+    @include H1;
+
     color: #1b344d;
-    font-size: 40px;
-    line-height: 51px;
     margin-bottom: 10px;
   }
 
   &__back-button {
+    @include Paragraph;
+
     margin-bottom: 18px;
     color: #22232c;
-    font-size: 16px;
-    line-height: 21px;
     position: absolute;
     top: -25px;
   }
 
   &__secret-phrase {
+    @include Paragraph;
+
     height: 190px;
     width: 495px;
     border: 1px solid #cdcdcd;
     border-radius: 6px;
     background-color: #fff;
     padding: 17px;
-    font-size: 16px;
   }
 
   &__secret-phrase::placeholder {
     color: #9b9b9b;
     font-weight: 200;
   }
 
-  &__faq-link {
-    font-size: 18px;
-    line-height: 23px;
-  }
-
   &__selector-label {
-    color: #1b344d;
-    font-size: 16px;
-  }
+    @include Paragraph;
 
-  &__dropdown {
```

Comment:
Are this and many others simply not used?

Reply:
Yes, they seem to be older styles that are no longer used.